.github/workflows/send-screenshots-to-pixeleagle.yml

```diff
@@ -0,0 +1,95 @@
+name: Send Screenshots to Pixel Eagle
+
+on:
+  workflow_call:
+    inputs:
+      artifact:
+        required: true
+        type: string
+      commit:
+        required: true
+        type: string
+      branch:
+        required: true
+        type: string
+      os:
+        required: true
+        type: string
+
+jobs:
+  send-to-pixel-eagle:
+    name: Send screenshots to Pixel Eagle
+    # runs on macos because it has the updated curl that will accept the --json flag, and the script is probably not portable to windows
+    runs-on: macos-14
+    steps:
+      
+      - name: Download artifact
+        uses: actions/download-artifact@v4
+        with:
+          pattern: ${{ inputs.artifact }}
+      
+      - name: Send to Pixel Eagle
+        run: |
+          project="B04F67C0-C054-4A6F-92EC-F599FEC2FD1D"
```

Comment:
Can this be set using the `env` property? I feel like it signals better that this is a hardcoded constant, and may be changed in the future.

```yaml
- name: Send to Pixel Eagle
  env:
    project: B04F67C0-C054-4A6F-92EC-F599FEC2FD1D
  run: ...
```